pkg/trace/stats/aggregation.go

```diff
@@ -29,14 +32,15 @@ type Aggregation struct {
 
 // BucketsAggregationKey specifies the key by which a bucket is aggregated.
 type BucketsAggregationKey struct {
-	Service     string
-	Name        string
-	PeerService string
-	Resource    string
-	Type        string
-	SpanKind    string
-	StatusCode  uint32
-	Synthetics  bool
+	Service      string
+	Name         string
+	PeerService  string
```

Comment:
Are we wanting this to still be a separate tag?

Reply:
It's a fair question. I don't think we have to put it with the rest of `PeerTags` today but that could be a future change.

Reply:
Also: the `peer_tags` field is meant to be an extra config opt-in. It would be weird to have it be "on" anyway because of `peer.service`?